src/ep.config.json

```diff
@@ -40,15 +40,6 @@
       "path": "",
       "pathForProxy": "",
       "reqTimeout": "30000"
-    },
```

Comment:
Are we still using b2b.authServiceAPI anywhere? I thought with the new endpoints we shouldn’t need it anymore